getting_started/first_steps.md

```diff
@@ -146,19 +146,23 @@ One of the most common usecases for Deno is building an HTTP Server.
 
 ```ts
 import { serve } from "https://deno.land/std@0.157.0/http/server.ts";
+import { urlJoin } from 'https://deno.land/x/url_join/mod.ts';
 
-const port = 8080;
+const port = 8000;
```

Comment:
Hah, came here to fix this too.  😅 

From what i could tell though, `port` isn't used at all, so I think it could just be deleted?  It seems like `serve` comes with its own default value for it.